### PR TITLE
Add a `confettiSource` prop to specify where the confetti spawns

### PR DESCRIPTION
**Pull request by balderdash**
Thanks for accepting my last two PRs! I have one more for you:

Add a prop `confettiSource` that specifies where the particles spawn, specified as an object with `x`/`y`/`w`/`h` keys. Here's the demo page with a `confettiSource` of `{x: 500, y: 200, w: 50, h: 50}`:
screenshot from 2017-08-29 08-27-17.

### CODE REVIEW

**Comment by balderdash** [comment]
@alampros Friendly ping. Just let me know if you want me to make any changes.

**Comment by alampros** [comment]
Oooh – very nice feature! Thanks!

File: src/confetti.js [hunk 1]
```diff
@@ -19,7 +25,6 @@ function confetti(canvasObj) {
   let recycle = true
 
   function self() {
-    const canvas = canvasObj
     const context = canvas.getContext('2d')
```
Comment (alampros) [inline src/confetti.js]: Shouldn't this be `canvasObj`?

Comment (alampros) [reply src/confetti.js]: Oh my. I should really have my coffee before I attempt to read code.

**Comment by balderdash** [comment]
Thanks for accepting! Would you be able to publish a new version of the npm package with these changes?

**Comment by alampros** [comment]
Of course - will publish v2.0.0 with a few other features in a bit

**Comment by alampros** [comment]
Published in `v2.0.0`

**Pull request merged**